sphinx/events.py

```diff
@@ -45,6 +45,7 @@ class EventListener(NamedTuple):
     'warn-missing-reference': 'domain, node',
     'doctree-resolved': 'doctree, docname',
     'env-updated': 'env',
+    'write-starting': 'builder',
```

Comment:
To be consistent with the other names, I would have chosen `write-started` or `builder-write`

Reply:
changed to `write-started`